src/components/Onboarding/PageIndicator.tsx

```diff
@@ -0,0 +1,98 @@
+import styled from '@emotion/styled';
+import React from 'react';
+import Image from 'next/image';
+import starIcon from '@assets/star.svg';
+import { useRouter } from 'next/router';
+
+interface PageProps {
+  index: Number;
+}
+
+function PageIndicator({ index }: PageProps) {
+  const router = useRouter();
+
+  const handleSkipClick = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+    e.preventDefault();
+    router.push('/start');
+  };
+
+  const handleNextClick = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+    e.preventDefault();
+    if (index < 2) {
+      let nextArrow = document.getElementsByClassName('slick-next')[0] as HTMLElement;
+      nextArrow.click();
+    } else {
+      router.push('/start');
+    }
+  };
+
+  return (
+    <Styled.Root>
+      <Styled.SkipButton onClick={handleSkipClick}>건너뛰기</Styled.SkipButton>
+      <Styled.PageStarWrapper>
+        <Styled.FirstStar index={index}>
+          <Image src={starIcon} alt="first" />
+        </Styled.FirstStar>
+        <Styled.SecondStar index={index}>
+          <Image src={starIcon} alt="second" />
+        </Styled.SecondStar>
+        <Styled.ThirdStar index={index}>
+          <Image src={starIcon} alt="third" />
+        </Styled.ThirdStar>
+      </Styled.PageStarWrapper>
+      <Styled.NextButton onClick={handleNextClick}>다음</Styled.NextButton>
+    </Styled.Root>
+  );
+}
+
+export default PageIndicator;
+
+const Styled = {
+  Root: styled.div`
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 50px;
+    position: fixed;
+    bottom: 0px;
+    padding: 0px 35px 0px 23px;
+    width: 600px;
+    @media (max-width: 767px) {
+      width: 414px;
+    }
+    @media (max-width: 413px) {
+      width: 100%;
+    }
+  `,
+  SkipButton: styled.button`
+    font-size: 12px;
+    color: #ffffff;
+  `,
+  PageStarWrapper: styled.div`
+    display: flex;
+    justify-content: space-between;
+    width: 20vw;
+    max-width: 150px;
+    height: 33px;
+
+    & * {
+      width: 25px;
+      height: 33px;
+    }
+  `,
+
+  FirstStar: styled.div<{ index: Number }>`
+    ${(props) => (props.index == 0 ? { opacity: 1 } : { opacity: 0.6 })}
+  `,
+  SecondStar: styled.div<{ index: Number }>`
+    ${(props) => (props.index == 1 ? { opacity: 1 } : { opacity: 0.6 })}
+  `,
+  ThirdStar: styled.div<{ index: Number }>`
+    ${(props) => (props.index == 2 ? { opacity: 1 } : { opacity: 0.6 })}
+  `,
```

Comment:
\* -  selector 개인적으로 조금 위험한것 같음!
안에 있는 모든것에 대해서 적용하는 것이니까,
1. 불필요하게 적용될 수 있다.
2. 확장성이 부족하다. 다른것들이 추가되었을 때 대응하기 어렵다.

그래서 하나하나 지정해주는 것이 좋아보임!
그리고 여기서는 Image에 대해서만 크기를 지정해주면 되니까 Image 태그 안에서 width 와 height를 지정해주자!

각각의 star에 props를 내려주고 있는데, 상위 엘리먼트에 index를 내려주고, nth-of-type을 이용하여 동적으로 할당하면 되겠습니다!

FirstStar, SecondStar 얘들 스타일드 컴포넌트 안쓰고 걍 div로 해줘도 될듯

```suggestion
  PageStarWrapper: styled.div<{ index: number }>`
    display: flex;
    justify-content: space-between;
    width: 20vw;
    max-width: 150px;
    height: 33px;

    & > div {
      opacity: 0.6;
    }

    & > div:nth-of-type(${({ index }) => index + 1}) {
      opacity: 1;
    }
```